frontend/packages/console-app/src/components/nodes/NodeTerminal.tsx

```diff
@@ -115,7 +116,8 @@ const NodeTerminalInner: React.FC<NodeTerminalInnerProps> = ({ obj }) => {
           error={
             <>
               The debug pod failed.{' '}
-              {obj?.data?.status?.containerStatuses[0]?.state?.terminated?.message}
+              {_.get(obj, 'data.status.containerStatuses[0].state.terminated.message') ||
```

Comment:
Prefer optional chaining to `_.get`

```suggestion
              {obj?.data?.status?.containerStatuses[0]?.state?.terminated?.message ||
```